protected/humhub/modules/tour/widgets/views/tour.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+use humhub\components\View;
+use humhub\helpers\Html;
+use humhub\modules\tour\TourConfig;
+
+/**
+ * @var $this View
+ * @var $config array
+ */
+
+?>
+
+<script <?= Html::nonce() ?>>
```

Comment:
Maybe we could remove this view file and directly render the JS code in the widget?

Reply:
Good idea!
Done in commit https://github.com/humhub/humhub/pull/7535/commits/c6866af10c6e767bcc99b4219ca9a7f06a8e5817